security/security_profiles_operator/spo-release-notes.adoc

```diff
@@ -12,6 +12,24 @@ These release notes track the development of the Security Profiles Operator in {
 
 For an overview of the Security Profiles Operator, see xref:../../security/security_profiles_operator/spo-overview.adoc#[Security Profiles Operator Overview].
 
+[id="spo-release-notes-0-8-1"]
+== Security Profiles Operator 0.8.1
+
+The following advisory is available for the Security Profiles Operator 0.8.1:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5958[RHBA-2023:5958 - OpenShift Security Profiles Operator bug fix update]
+
+[id="spo-0-8-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, `SELinuxProfiles` did not inherit custom `SELinuxProfiles` from the same namespace. With this update, the issue has now been resolved and `SELinuxProfiles` are inherited from the same namespace as expected. (link:https://issues.redhat.com/browse/OCPBUGS-17164[*OCPBUGS-17164*])
+
+* Previously, RawSELinuxProfiles would hang during the creation process and would not reach an `Installed` state. With this update, the issue has been resolved and RawSELinuxProfiles are created successfully. (link:https://issues.redhat.com/browse/OCPBUGS-19744[*OCPBUGS-19744*])
+
+* Previously, patching the `enableLogEnricher` to `true` would cause the `seccompProfile` `log-enricher-trace` pods to be stuck in a `Pending` state. With this update, `log-enricher-trace` pods reach an `Installed` state as expected. (link:https://issues.redhat.com/browse/OCPBUGS-22182[*OCPBUGS-22182*])
+
+* Previously, the Security Profiles Operator generated high cardinality metrics, causing Prometheus pods using high amounts of memory. With this update, cardinality is reduced and the Prometheus pods use less memory. (link:https://issues.redhat.com/browse/OCPBUGS-22406[*OCPBUGS-22406*])
```

Comment:
Not sure "cardinality is reduced" is accurate or not. With this date, the below 3 metrics will disappear for the SPO namespace.
rest_client_request_duration_seconds
rest_client_request_size_bytes
rest_client_response_size_bytes
![Screenshot from 2023-12-14 14-21-58](https://github.com/openshift/openshift-docs/assets/55695225/d3c1c2ff-b2cc-4359-8273-d0cf78a39c51)

Reply:
Done: 
![image](https://github.com/openshift/openshift-docs/assets/46728411/51da3d37-0b50-4257-a093-32a63935cd70)